### DISCUSSION TITLE
InfluxDB interval disassociate with teleperiod

### DISCUSSION THREAD

**Discussion by pkkrusty** [arendst/Tasmota#13745] [Q&A] [answered]
I'm on 10.0.0.3 and loving the influxdb integration. However, for lots of things, like energy monitoring and certain sensors, preference is to update as often as possible for granular data. But this tends to clog up the console and mqtt and syslog. Is there a way to have a new SetOption which, if defined as 0 would follow the teleperiod time interval for InfluxDB updates, but set from 1->largeNumber would indicate how many second interval InfluxDB sends data? Default to 0.

**Comment by barbudor** [comment, marked as answer]
For more granular energy monitoring, I'm using PowerDelta which allows to trigger telemetry as soon as a (configurable) change is detected.
Although I'm not using the native influxdb driver (I'm doing it from node-red), I would expect it also works with influxdb driver

**Reply by barbudor** [reply]
Looking into the InfluxDB driver's code, it seems he has it's own period management which is not synchronized with Teleperiod.
So PowerDelta should probably have no impact right now.
Need to dig further and setup a test platform before proposing a change.

**Reply by barbudor** [reply]
https://github.com/arendst/Tasmota/pull/13750

**Reply by pkkrusty** [reply]
Trying this out now, and seems to be working correctly. Noted yesterday that for simple state changes (like `Power1 1` or `Power1 0`, InfluxDB driver seems to catch and send those, regardless of teleperiod. Not sure if you had already checked the driver with PowerDelta, but PowerDelta doesn't work with PZEM-004T(I got command unknown when I typed it), so I couldn't mess around with it.

**Reply by barbudor** [reply]
There is a special processing for power states so they are reported immediately.
So if powerdelta doesn't work with you, I'm feeling less guilty to went the easy way :)

**Reply by pkkrusty** [reply]
Sure helped me out, thanks again!
